dev-docs/publisher-api-reference/setConfig.md

```diff
@@ -1404,6 +1404,18 @@ pbjs.setConfig({
 
 <a name="setConfig-Generic-Configuration"></a>
 
+### Enable Previous Auction Info
```

Comment:
typically configuration that requires a particular module is not listed here but on that module's page.